packages/esm-patient-programs-app/src/programs/programs-form.workspace.tsx

```diff
@@ -38,6 +39,7 @@ const createProgramsFormSchema = (t: TFunction) =>
     enrollmentDate: z.date(),
     completionDate: z.date().nullable(),
     enrollmentLocation: z.string(),
+    selectedState: z.string(),
   });
 
```

Comment:
Then, we could fetch the Config schema here using the `useConfig` hook like so:

```tsx
const config = useConfig();
```

and then we can scope all the Program status logic via the boolean property we set earlier in the schema:

```tsx
config.showProgramStatusField && (
	style: { width: '50%' },
     legendText: '',
     value: stateSelect,
	}
)

etc...